release-please-config.json

```diff
@@ -10,7 +10,7 @@
     "packages/sdk-node": {},
     "packages/sdk": {},
     ".": {
-      "release-as": ""
+      "release-as": "1.1.0"
```

Comment:
packages/sdk should be "21.4.0" or "21.4.1" shouldn't it?

Reply:
so, this config file is a little tricky. L4 is setting the default for every package. Then each package can override it with either a specific release-as version or `""` which means "override with whatever conventional commits would say"

That's what `"."` and `"packages/sdk-codegen"` were set to but then I goofed it up and leaked a "BREAKING CHANGE" into their history (really the breaking change is only for the python SDK but I included the sdk-codegen changes in that single commit which I really should not have done given these consequencyes). So temporarily we have to set them to just bump the minor version like we actually want them to.

After #532 eventually merges we'll set these back to to `""`

Reply:
All righty then! Glad you can wrap your head around this.

Reply:
open to design feedback. Our use-case is complicated with a mix of semver and non-semver packages as well as the inter-connectedness of packages and how that relates to forming conventional-commits/PRs that then correspond to version bumps and changelog entries. I'm writing a doc to hopefully distribute that knowledge so I'm not the only key holder